chapter4.txt

```diff
@@ -17,7 +17,7 @@ Series is like numpy's array/dictionary, though it comes with a lot of extra fea
 
 You should get Pandas if you installed Anaconda on Windows. On Linux, run
 
-```
+``` bash
```

Comment:
While you are technically correct , in adding bash/python in front of the coding block, this will need to be removed when I add the text to my wordpress blog. So can you please remove all these?